src/renderers/shared/fiber/ReactFiberScheduler.js

```diff
@@ -1181,6 +1186,9 @@ module.exports = function<T, P, I, TI, PI, C, CX, PL>(
       // The risk is that the return path from this Fiber may not be accurate.
       // That risk is acceptable given the benefit of providing users more context.
       const componentStack = getStackAddendumByWorkInProgressFiber(failedWork);
+      const componentStackFrames = getStackFramesByWorkInProgressFiber(
```

Comment:
I considered mapping `componentStackFrames` through `describeComponentFrame` for the `componentStack` (rather than re-crawling the graph) but decided it wasn't worth it given that we don't hit this code path often.